include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp

```diff
@@ -10,6 +10,8 @@
 #include <rofl/common/openflow/cofflowmod.h>
 #include <rofl/common/openflow/cofgroupmod.h>
 
+#define OFPCML_NO_BUFFER 0xffff
```

Comment:
this is already defined in rofl-comon, could we use that? e.g. src/rofl/common/openflow/openflow13.h:652